src/types/inventory/stackrequest/CraftRecipeAutoStackRequestAction.php

```diff
@@ -53,19 +50,19 @@ public function getIngredients() : array{ return $this->ingredients; }
 
 	public static function read(PacketSerializer $in) : self{
 		$recipeId = $in->readRecipeNetId();
-		$repetitions2 = $in->getByte();
 		$repetitions = $in->getByte();
+		$in->getByte(); //repetitions property is sent twice, mojang...
```

Comment:
Don't do this. Even if it is currently sent twice, both copies should be retained. If it starts sending something different in the future this will quietly break and no one will be any the wiser.

Reply:
That being said I don't love `repetitions2` coming before `repetitions`...